docs/usage.md

```diff
@@ -193,18 +197,18 @@ A location does not have a prescribed shape. In your UI it can represent a singl
 
 The location property will be set on the [member](#members).
 
-Because locations are part of members, a `memberUpdate` event will be emitted when a member updates their location. When a member leaves, their location is set to `null`.
+Because locations are part of members, an `update` member event will be emitted when a member updates their location. When a member leaves, their location is set to `null`.
```

Comment:
`update` is an optional event name. The subscription will also be triggered with no event name. So we can just say "Because locations are part of members, a member event will be emitted"

Reply:
I don't think this is true anymore - this is applicable to `space`, not `space.members`